### PR TITLE
show Aeron stats in tests

### PR DESCRIPTION
**Pull request by patriknw**
Show the Aeron stats counters. Can be useful when tuning/debugging.

Also updated to Aeron 0.9.7 and change the number format to more readable `650,342 msgs/sec`

The counters look like:

```
[JVM-2] receiver stats:
[JVM-2]   0:                    0 - Bytes Sent
[JVM-2]   1:          331,120,696 - Bytes received
[JVM-2]   2:                    0 - Failed offers to ReceiverProxy
[JVM-2]   3:                    0 - Failed offers to SenderProxy
[JVM-2]   4:                    0 - Failed offers to DriverConductorProxy
[JVM-2]   5:                   52 - NAKs sent
[JVM-2]   6:                    0 - NAKs received
[JVM-2]   7:                6,365 - Status Messages sent
[JVM-2]   8:                    0 - Status Messages received
[JVM-2]   9:                    0 - Heartbeats sent
[JVM-2]  10:                   10 - Heartbeats received
[JVM-2]  11:                    0 - Retransmits sent
[JVM-2]  12:                    0 - Flow control under runs
[JVM-2]  13:                    0 - Flow control over runs
[JVM-2]  14:                    0 - Invalid packets
[JVM-2]  15:                    0 - Errors
[JVM-2]  16:                    0 - Data Packet short sends
[JVM-2]  17:                    0 - Setup Message short sends
[JVM-2]  18:                    0 - Status Message short sends
[JVM-2]  19:                    0 - NAK Message short sends
[JVM-2]  20:                    4 - Client keep-alives
[JVM-2]  21:                    0 - Sender flow control limits applied
[JVM-2]  22:                    0 - Unblocked Publications
[JVM-2]  23:                    0 - Unblocked Control Commands
[JVM-2]  24:                    0 - Possible TTL Asymmetry
[JVM-2]  25:          100,764,016 - Rcv-hwm: 9 -771653916 10 aeron:udp?endpoint=localhost:20512
[JVM-2]  26:          128,000,896 - Rcv-hwm: 2 -771653918 10 aeron:udp?endpoint=localhost:20512
[JVM-2]  27:          102,406,144 - Rcv-hwm: 5 -771653917 10 aeron:udp?endpoint=localhost:20512
[JVM-2]  29:          128,000,896 - Rcv-hwm: 6 -771653918 10 aeron:udp?endpoint=localhost:20512
[JVM-2]  31:          128,000,896 - Rcv-hwm: 10 -771653918 10 aeron:udp?endpoint=localhost:20512
[JVM-2]  33:          102,406,144 - Rcv-hwm: 11 -771653917 10 aeron:udp?endpoint=localhost:20512
```

### CODE REVIEW

**Comment by drewhk** [comment]
LGTM, looks nice!

The 52 NAKs is interesting. I guess it is UDP buffer drops.

File: akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala [hunk 1]
```diff
@@ -80,6 +82,9 @@ abstract class AeronStreamLatencySpec
 
   val driver = MediaDriver.launchEmbedded()
 
+  val stats =
+    new AeronStat(AeronStat.mapCounters(new File(driver.aeronDirectoryName, CncFileDescriptor.CNC_FILE)))
```
Comment (ktoso) [inline akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala]: what does CNC stand for?

Comment (patriknw) [inline akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala]: "command and control", see description of its layout in io.aeron.CncFileDescriptor

Comment (ktoso) [inline akka-remote-tests/src/multi-jvm/scala/akka/remote/artery/AeronStreamLatencySpec.scala]: Will do, thanks

**Comment by ktoso** [comment]
LGTM :)

**Pull request merged**